File: aoj/volume0/n0016/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in).useDelimiter(",|\\n");

        double x = 0;
        double y = 0;

        int theta = 90;

        while (true) {

            int distance = sc.nextInt();
            int angle = sc.nextInt();

            if (distance == 0 && angle == 0) {
                break;
            }

            x += distance * Math.cos(Math.toRadians(theta));
            y += distance * Math.sin(Math.toRadians(theta));

            theta -= angle;

        }

        System.out.println((int) x);
        System.out.println((int) y);

    }

}

File: aoj/volume0/n0016/Main.cpp
#include <iostream>
#include <cstdio>
#include <cmath>

using namespace std;

int main() {

    double x = 0;
    double y = 0;

    int theta = 90;

    while (1) {

        int distance, angle;
        scanf("%d,%d", &distance, &angle);

        if (distance == 0 && angle == 0) {
            break;
        }

        double radius = theta * M_PI / 180.0;

        x += distance * cos(radius);
        y += distance * sin(radius);

        theta -= angle;

    }

    cout << (int) x << endl << (int) y << endl;

    return 0;

}
